posts/2025-02-17_theres_a_pharmaverse_package_for_that/managing-clinical-trial-data.qmd

```diff
@@ -0,0 +1,329 @@
+      AGE            AGEU               SEX                RACE              ETHNIC             ARMCD          
+ Min.   :50.00   Length:306         Length:306         Length:306         Length:306         Length:306        
+ 1st Qu.:70.25   Class :character   Class :character   Class :character   Class :character   Class :character  
+ Median :77.00   Mode  :character   Mode  :character   Mode  :character   Mode  :character   Mode  :character  
+ Mean   :75.09                                                                                                 
+ 3rd Qu.:81.00                                                                                                 
+ Max.   :89.00                                                                                                 
+                                                                                                               
+     ARM              ACTARMCD            ACTARM            COUNTRY             DMDTC                DMDY    
+ Length:306         Length:306         Length:306         Length:306         Length:306         Min.   :-37  
+ Class :character   Class :character   Class :character   Class :character   Class :character   1st Qu.:-14  
+ Mode  :character   Mode  :character   Mode  :character   Mode  :character   Mode  :character   Median :-10  
+                                                                                                Mean   :-11  
+                                                                                                3rd Qu.: -7  
+                                                                                                Max.   : -2  
+                                                                                                NA's   :52   
+    TRT01P             TRT01A              EOSDT               SAFFL          
+ Length:306         Length:306         Min.   :2012-09-01   Length:306        
+ Class :character   Class :character   1st Qu.:2013-05-16   Class :character  
+ Mode  :character   Mode  :character   Median :2013-09-30   Mode  :character  
+                                       Mean   :2013-10-15                     
+                                       3rd Qu.:2014-03-20                     
+                                       Max.   :2015-03-05                     
+                                       NA's   :52                             
+[1] "\nADSL has been saved to 'adsl_output.csv'"
+```
+
+#### **More Details on ADSL Creation**
+
+This is just a **high-level example**; the full process includes deriving death variables, grouping populations, and applying labels. For a deeper dive, check out the [ADSL Implementation Guide](https://cran.r-project.org/web/packages/admiral/vignettes/adsl.html).
+
+![](images/adsl.png)
```

Comment:
Umm...I think this image can just be dropped - the link is enough

Reply:
dropped